components.json

```diff
@@ -351,7 +352,7 @@
 		"haml": {
 			"title": "Haml",
 			"require": "ruby",
-			"peerDependencies": [
+			"after": [
```

Comment:
I know I'm coming back to this after a while but `after` doesn't really say what this is supposed to do. Maybe this should be `optionalDependencies`? Is that what this means?

Reply:
Yes, it's an optional dependency. I used `after` because it was already used by some plugins.

Also, regarding `optionalDependencies`: I'm not a fan of the `Dependencies` suffix. This is why I choose simple one-word names.

Reply:
Gonna resolve this convo and focus the bikeshedding the names in the other thread.